src/app/collections/PresentationNode.tsx

```diff
@@ -26,12 +33,28 @@ interface Props {
       visible: number;
     };
   };
+  settings: Settings;
+  setSetting;
```

Comment:
The pattern we've been using for Redux props is:

* `ProvidedProps` for stuff that you set on the component when using it.
* `StoreProps` for stuff that comes from `mapStateToProps`.
* `DispatchProps` for stuff from `mapDispatchToProps`.

Then at the end there's the `type Props = ProvidedProps & StoreProps & DispatchProps`. Not only does this make it clearer where everything comes from, it helps to narrow the typechecking for stuff like `mapStateToProps` so that it's easier to know when things aren't right.